cloudify_agent/shell/commands/configure.py

```diff
@@ -49,8 +45,58 @@ def configure(disable_requiretty, relocated_env, no_sudo):
         click.echo('Disabling requiretty directive in sudoers file')
         _disable_requiretty(no_sudo)
         click.echo('Successfully disabled requiretty for cfy-agent')
-    if relocated_env:
-        _relocate_venv()
+    if fix_shebangs:
+        click.echo('Fixing shebangs in scripts')
+        _fixup_scripts()
+
+
+def _fixup_scripts():
+    """Make scripts in bin_dir relative by rewriting their shebangs
+
+    Examine each file in bin_dir - if it looks like a python script, and has a
+    shebang - replace it with a shebang pointing to the agent's python.
+    """
+    from cloudify_agent.shell.main import get_logger
+    logger = get_logger()
+
+    new_shebang = f"#!{sys.executable}"
```

Comment:
we need to understand what `sys.executable` actually is. If it's `/opt/cloudify-agent-7.0.0/bin/python`, fine. But if it contains the agent node id in its path, then I don't think we can do this. Shebangs are limited to 128 bytes. This is tested by the "long path" test in this repo, so if that keeps failing, this would be why. Maybe we'll need something like virtualenv's relocatable. That'd be unfortunate.

Reply:
OK. We did the silly "multi-line shebang" trick